### PR TITLE
[MV-138] Add max bandwidth api in Android sdk

### PR DESCRIPTION
**Pull request by graszka22**

### CODE REVIEW

File: MembraneRTC/src/main/java/org/membraneframework/rtc/MembraneRTC.kt [hunk 1]
```diff
@@ -163,6 +158,27 @@ private constructor(
...
     }
 
+    /**
+     * Updates maximum bandwidth for the track identified by trackId.
+     * This value directly translates to quality of the stream and, in case of video, to the amount of RTP packets being sent.
+     * In case trackId points at the simulcast track bandwidth is split between all of the variant streams proportionally to their resolution.
+     * @param trackId - track id of a video track
+     * @param bandwidthLimit - bandwidth in kbps
+     */
+    public fun setTrackBandwidth(trackId: String, bandwidthLimit: TrackBandwidthLimit) {
```
Comment (mickel8) [inline MembraneRTC/src/main/java/org/membraneframework/rtc/MembraneRTC.kt:168]: To be compliant with js API this should be `TrackBandwidthLimit.BandwidthLimit`

**Review by mickel8 — CHANGES_REQUESTED** [review]

File: MembraneRTC/src/main/java/org/membraneframework/rtc/media/VideoParameters.kt [hunk 2]
```diff
@@ -10,82 +10,100 @@ data class Dimensions(val width: Int, val height: Int) {
...
+ *
+ * @property dimensions: specified width x height of the video
+ * @property maxBitrate: specifies maximum bitrate of video stream
+ * @property maxFps: specifies maximum frame rate of the video stream
+ * @property simulcastConfig: specifies the simulcast configuration, by default it's turned off
  * <p>
  * Contains a set of useful presets.
  */
-public data class VideoParameters(val dimensions: Dimensions, val encoding: VideoEncoding) {
+public data class VideoParameters(val dimensions: Dimensions, val maxBitrate: TrackBandwidthLimit, val maxFps: Int, val simulcastConfig: SimulcastConfig = SimulcastConfig()) {
```
[the remark this answers is not in the document]
Comment (graszka22) [reply MembraneRTC/src/main/java/org/membraneframework/rtc/media/VideoParameters.kt:21]: Yes, it's used but not here. Here `simulcastConfig` is a field in the class, not only an argument in constructor (see here how data class works in Kotlin: https://kotlinlang.org/docs/data-classes.html). We're accessing it later like this: `localTrack.videoParameters.simulcastConfig`

[the remark this answers is not in the document]
Comment (mickel8) [reply MembraneRTC/src/main/java/org/membraneframework/rtc/media/VideoParameters.kt:21]: Oh, I see, thank you!

**Review by mickel8 — APPROVED** [review]
Nicely done! :slightly_smiling_face:

**Pull request merged**